portability-types-transfer/src/main/java/org/dataportabilityproject/types/transfer/models/calendar/CalendarModelWrapper.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dataportabilityproject.types.transfer.models.calendar;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.google.common.collect.ImmutableList;
+import java.util.Collection;
+import org.dataportabilityproject.types.transfer.models.DataModel;
+
+/**
+ * A Wrapper for all the possible objects that can be returned by a calendar exporter.
+ */
+@JsonTypeName("CalendarModelWrapper")
+public class CalendarModelWrapper extends DataModel {
```

Comment:
The name wrapper throws me off occasionally. Maybe we can take advantage of this refactor to rename the top level entities that encapsulate all the stuff for a type of data.

CalendarDataToTransfer
CalendarDataModel (since it extends DataModel)
CalendarTransferData 
CalendarTransferModel 
CalendarData